src/pages/access_review/AppSummary.js

```diff
@@ -36,10 +38,20 @@ export const AppSummary = hh(class AppSummary extends React.Component {
     super();
     this.state = {
       generateRestrictions: this.generateRestrictions.bind(this),
-      translatedRestrictions: []
+      translatedRestrictions: [],
+      institution: ""
     };
   }
 
+  getInstitutionFromProfile = async () => {
+    const institute = await Institution.getById(this.props.researcherProfile.institutionId);
+    this.setState(prev => {
+      prev.institution = institute.name;
+      return prev;
+    });
+  };
```

Comment:
Here's another case where we should be getting the full institution from the user API instead of querying for it directly. So, not for this PR, but for the tech-debt one that should make this cleaner.